### PR TITLE
Update Recommendation to v2 of the bundle

### PR DESCRIPTION
**Pull request by DominikaK**
Modifies recommendation.md to cover v2 of Recommendation bundle.

(all current branches)

Based on doc provided by @johaho

Once this is merged, https://github.com/ezsystems/EzSystemsRecommendationBundle/blob/master/README.md will be replaced with a simple note linking to this page.

### CODE REVIEW

**Comment by DominikaK** [comment]
Ping @johaho for review

File: docs/guide/recommendation.md [hunk 1]
```diff
@@ -230,21 +252,297 @@ You can also bypass named arguments using standard value passing as arguments.
...
+
+After defining what Content Types should be recommended and tracked you can start the full export with the following command:
+
+``` bash
+php ezpublish/console ezreco:runexport --contentTypeIdList=16 --webHook=https://admin.yoochoose.net/api/<your_customer_id>/items --hidden=1 --mandatorId=<your_customer_id> --host=<your_ezplatform_host_with_scheme>
+```
+
+By running this command, the bundle exporter collects all content related to the SiteAccesses of this customer id and places it in files.
+After finishing, the systems sends a POST request to the `webHook` endpoint and informs the recommender how to fetch the data.
+The recommender triggers an internal workflow so that the generated files are downloaded and imported in the recommender content store.
```
Comment (MagdalenaZuba) [inline docs/guide/recommendation.md]: recommender's - consistent with the title below

File: docs/guide/recommendation.md [hunk 2]
```diff
@@ -230,21 +252,297 @@ You can also bypass named arguments using standard value passing as arguments.
...
+
+You can verify the import controller of the bundle by calling the local API. You should use the 'Accept' header and may need to add an Authorization header if some authentication is required.
+
+To check if the `content` interface is working as expected, try this URI:
+
+`GET http://{endpoint}/api/ezp/v2/ez_recommendation/v1/content/{contentId}`
+
+`Accept    application/vnd.ez.api.Content+json`
+
+Additionally you should check if the `contenttypes` interface is working as well by calling the following URI:
```
[the remark this answers is not in the document]
Comment (DominikaK) [reply docs/guide/recommendation.md]: It's spelt without capital in the actual code (two lines below)

File: docs/guide/recommendation.md [hunk 3]
```diff
@@ -230,21 +252,297 @@ You can also bypass named arguments using standard value passing as arguments.
...
+### 4. Check if the bundle provides REST data
+
+You can verify the import controller of the bundle by calling the local API. You should use the 'Accept' header and may need to add an 'Authorization' header if some authentication is required.
+
+To check if the `content` interface is working as expected, try this URI:
+
+`GET http://{endpoint}/api/ezp/v2/ez_recommendation/v1/content/{contentId}`
+
+`Accept    application/vnd.ez.api.Content+json`
+
```
Comment (damianz5) [inline docs/guide/recommendation.md]: Missing `Authorization    Basic xxxxxxxx`

File: docs/guide/recommendation.md [hunk 4]
```diff
@@ -230,21 +252,297 @@ You can also bypass named arguments using standard value passing as arguments.
...
+Additionally you should check if the `contenttypes` interface is working as well by calling the following URI:
+
+`GET http://{endpoint}/api/ezp/v2/ez_recommendation/v1/contenttypes/38?page=1&page_size=10`
+
+`Accept    application/vnd.ez.api.Content+json`
+
```
Comment (damianz5) [inline docs/guide/recommendation.md]: same as above

File: docs/guide/recommendation.md [hunk 5]
```diff
@@ -230,21 +252,297 @@ You can also bypass named arguments using standard value passing as arguments.
...
+    }
+}
+```
+
+## Running a full content export
+
+After defining what Content Types should be recommended and tracked you can start the full export with the following command:
+
+``` bash
+php ezpublish/console ezreco:runexport --contentTypeIdList=16 --webHook=https://admin.yoochoose.net/api/<your_customer_id>/items --hidden=1 --mandatorId=<your_customer_id> --host=<your_ezplatform_host_with_scheme>
```
Comment (damianz5) [inline docs/guide/recommendation.md]: Could also be: (Symfony 2.8) `php app/console` / (Symfony >=3.x) `php bin/console`
(three other occurrences in this PR)

File: docs/guide/recommendation.md [hunk 6]
```diff
@@ -122,74 +164,217 @@ ez_recommendation:
...
+
+Additionally you should check if the `contenttypes` interface is working as well by calling the following URI:
+
+`GET http://{endpoint}/api/ezp/v2/ez_recommendation/v1/contenttypes/38?page=1&page_size=10`
+
+`Accept    application/vnd.ez.api.Content+json`
+
+`Authorization    Basic xxxxxxxx`
+
+Both interfaces are supposed to provide content data in .json format.
```
Comment (mere-teresa) [inline docs/guide/recommendation.md]: _in JSON format_: it's the format, not a file

File: docs/guide/recommendation.md [hunk 7]
```diff
@@ -122,74 +164,217 @@ ez_recommendation:
...
+After finishing, the systems sends a POST request to the `webHook` endpoint and informs the personalization engine to fetch the data.
+An internal workflow is then triggered so that the generated files are downloaded and imported in the personalization engine's content store. Please be patient, as this can take up to a couple of minutes.
+
+!!! caution "Changing Content Types for recommendations"
+
+    If the Content Types to be recommended are changed, a full export needs to be started again by running the `php app/console ezreco:runexport` command.
+
+### Checking if the full context export was stored in the recommender engine
+
+There are three ways to check if content was transferred and stored successfully in the recommender engine.
```
Comment (mere-teresa) [inline docs/guide/recommendation.md]: Maybe list the 3 ways here, because titles are far away from each other.

**Comment by johaho** [comment]
Hi @DominikaK 
can you add that there's BASIC Auth needed in section https://github.com/ezsystems/developer-documentation/blob/8d43890af35b25b94cfd849cb5ccdb353d76b9bd/docs/guide/recommendation.md#rest-request-to-the-recommenders-content-store
The BASIC AUth username is the customerID and the password is the licensekey. 

The other two do not need Auth

**Pull request merged**